src/tools/annotation/bidirectionalTool/moveHandle/moveLongLine/getDistanceToIntersection.js

```diff
@@ -0,0 +1,21 @@
+import getDistanceWithPixelSpacing from '../../utils/getDistanceWithPixelSpacing';
+
+/**
+ * Returns the distance between the intersection and the given point
+ * considering the row and column pixel spacings.
+ *
+ * @param {*} baseData Base data for bidirectional line moving
+ * @param {*} point Point which the distance to intersection will be calculated
+ *
+ * @returns {number} Returns the distance betwen the intersection and point
+ */
+export default function getDistanceToIntersection(baseData, point) {
+  const { columnPixelSpacing, rowPixelSpacing, intersection } = baseData;
```

Comment:
The constructor doesn't need to take the entire `baseData` object if we're only using these three values from it.

Reply:
Follow-up question. Why does this method exist? It looks like all it does is map data to call a different utility function? I don't know that it adds any value.

Reply:
This function was doing something else then I refactored and that changed a bit.
It was behaving basically as a proxy function.
Then I removed it.